README.md

```diff
@@ -1,19 +1,81 @@
 # etl-pipeline
 
-
 ![Luigi Pipeline Scheduled](https://github.com/swb-ief/etl-pipeline/workflows/LuigiPipelineSchedule/badge.svg)
 
 # Structure [ Phase 2]
+
+Work in progress: This list should contain only that what we need for phase 2 everything else can be deleted or first
+moved to phase 1 archive
+
 ```
-|- backend [Main package with all the calculations]
-|- tests [Unit tests]
+│   .gitignore
+│   pytest.ini                                  [ Unittest configuration file ]
+│   README.md                                   [ This is me ]
+│   requirements.txt
+│
+├───.github
+│   └───workflows
+│           code_quality.yml                    [ Code quality validation on push and pull requests ]
+│           update_dashboard.yml                [ Main action to update the dashboard ]
+├───backend                                     [ Main package with all the code ]
+│   │   __init__.py
+│   │
+│   ├───data                                    [ data related classes and methods ]
+│   │   │   extract_covid19_india_data.py
+│   │   │   __init__.py
+│   │   │
+│   │   └───__pycache__
+│   │           extract_covid19_india_data.cpython-38.pyc
+│   │
+│   ├───metrics                                 [ covid 19 related metrics classes and methods ]
+│   │   │   calculations.py
+│   │   │   __init__.py
+│   │
+│   ├───repository                              [ repository related classes and methods ]
+│   │       gsheet_repository.py
+│   │       repository.py
+│   │       __init__.py
+│
+├───phase 1 archive                             [ Temporarey storage for files no longer needed by both P1 & P2 ]
+├───tasks                                       [ Luigie tasks executed by github workflow actions ]
+│   │   fetch_covid19_india_data_task.py
+│   │   update_gsheet_task.py
```

Comment:
do we want to add a task here for the ward data/metrics related processes: 
1) run pdf_scraper 
2) calculate ward level metrics 
3) update google sheet

Reply:
yes and no.
1. pdf scraper - Probably although we might move away from that in phase 2 if they can provide us soon with a google sheet access
2. Ward calculations no. they are identical for state and district. However collecting the ward data from all the different cities might become a task per city for each city might have a different way to provide the data. See Slack discussions on this topic.

IMHO the only benefit of luigi and its tasks is 1) retries and 2) parallelism of slow running tasks (keeping in mind the 2 vCPU cores we have on our github worker. So making tasks for downloads etc is perfect... for calculations we get little to no benefit nor do we expect any failures that benefit from a retry..